write-fonts/src/graph.rs

```diff
@@ -80,7 +80,6 @@ impl ObjectStore {
     }
 }
 
-#[derive(Debug)]
 pub struct Graph {
```

Comment:
I suggest a comment wrt why we don't derive Debug, otherwise it may be added back